python/ray/import_thread.py

```diff
@@ -30,13 +32,19 @@ class ImportThread(object):
         redis_client: the redis client used to query exports.
         threads_stopped (threading.Event): A threading event used to signal to
             the thread that it should exit.
+        imported_collision_identifiers: This is a dicitonary mapping strings
+            containing the collision identifier of the remote functions that
```

Comment:
`strings containing the collision identifier of the remote functions that have been exported` -> `collision identifiers for exported remote functions`